src/views/Teams/__snapshots__/Team.test.js

```diff
@@ -0,0 +1,12 @@
+import { render } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom/cjs/react-router-dom.min';
+import Team from '../../../views/Teams/Team';
+
+test('renders title message', () => {
+  const container = render(
```

Comment:
Making this test async and awaiting with findByText would allow the Component's useEffect to run and render all of the data.